src/transformers/onnx/config.py

```diff
@@ -56,6 +72,7 @@ class OnnxConfig(ABC):
 
     DEFAULT_FIXED_BATCH = 2
     DEFAULT_FIXED_SEQUENCE = 8
+    TORCH_ONNX_MINIMUM_VERSION = version.parse("1.8")
```

Comment:
I've picked the default minimum `torch` version to be v1.8 because that's what is defined in the `export()` function via `is_torch_onnx_dict_inputs_support_available()`

Reply:
I had to pause several times on this name, which is a global constant name.
All those class variables should have camel-cased name to be consistent with the rest of Transformers and not give a heart attack to a reviewer seeing a specific model change the value of a global constant.